src/native/external/zlib-version.txt

```diff
@@ -1,11 +1,12 @@
-v1.2.12
-(21767c654d31d2dccdde4330529775c6c5fd5389)
+v1.2.13
+(04f42ceca40f73e2978b50e93806c2a18c1281fc)
```

Comment:
I can see 04f4 is the latest release in madler/zlib: https://github.com/madler/zlib/releases